### PR TITLE
Cli docs test

### PR DESCRIPTION
**Pull request by thisisjofrank**

### CODE REVIEW

File: runtime/manual/tools/test.md [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+
+- `--allow-hrtime`
+    Allow [high-resolution time measurement](https://deno.land/manual/basics/permissions). Note: this can enable timing attacks and fingerprinting.
+
+- `--deny-hrtime`
+    Deny [high-resolution time measurement](https://deno.land/manual/basics/permissions). Note: this can prevent
+  timing attacks and fingerprinting.
+
+- `-A, --allow-all`
+    Allow [all permissions](https://deno.land/manual/basics/permissions).
```
Comment (ry) [inline runtime/manual/tools/test.md:185]: I think all these permission flags should be in their own section.

**Review by ry — APPROVED** [review]
otherwise lgtm

File: runtime/manual/tools/test.md (hunk 1 above)
Comment (thisisjofrank) [reply runtime/manual/tools/test.md:185]: I was thinking the same, but couldn't quite decide where to put them. I'll update now.

Comment (thisisjofrank) [reply runtime/manual/tools/test.md]: I've moved them into their own accordion for now.

**Pull request merged**